sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -102,7 +102,7 @@ public void onEvent(
             } else if (DeployGateEvent.ACTION_ONESHOT_LOGCAT.equals(action)) {
                 String captureId = null;
 
-                if (mDeployGateClient.isSupported(Compatibility.DEVICE_CAPTURE)) {
+                if (mHostApp.canUseDeviceCapture() && mDeployGateClient.isSupported(Compatibility.DEVICE_CAPTURE)) {
```

Comment:
SDK cannot turn off the device capture feature completely because triggers will be fired from the client app.